docs/release-notes/NuGet-5.7.md

```diff
@@ -0,0 +1,94 @@
+---
+title: NuGet 5.7 Release Notes
+description: Release notes for NuGet 5.7 including new features, bug fixes, and DCRs.
+author: chgill-msft
+ms.author: chgill
+ms.date: 8/14/2020
+ms.topic: conceptual
+---
+
+# NuGet 5.7 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**5.7.0**](https://nuget.org/downloads) | [Visual Studio 2019 version 16.7](https://visualstudio.microsoft.com/downloads/) | [3.1.401](https://dotnet.microsoft.com/download/dotnet-core/3.1)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio 2019 with .NET Core workload
+
+## Summary: What's New in 5.7
+
+### Features added in this release
+
+* Made noop restore faster - sped up evaluations by calling MSBuild Static Graph apis (dotnet.exe) - [#9644](https://github.com/NuGet/Home/issues/9644)
+
+* Added integration of PackageVersion information for non-sdk style projects in VS restore  - [#9236](https://github.com/NuGet/Home/issues/9236)
+
+* nuget.exe update -self -Source https://feed - [#1783](https://github.com/NuGet/Home/issues/1783)
+
+* Added support for multiple config files in %APPDATA%\NuGet directory - [#9394](https://github.com/NuGet/Home/issues/9394)
+
+* DeterministicSourcePaths now takes NuGet source packages into account - [#9431](https://github.com/NuGet/Home/issues/9431)
+
+* Added extern alias support for NuGet package references - [#4989](https://github.com/NuGet/Home/issues/4989)
+
+* Added INuGetProjectService.GetInstalledPackagesAsync extensibility API - [#9702](https://github.com/NuGet/Home/issues/9702)
+
+* Added interop API to enumerate fallback folders without requiring a solution/project - [#9395](https://github.com/NuGet/Home/issues/9395)
+
+* Added "latest" option for -MSBuildVersion - [#8808](https://github.com/NuGet/Home/issues/8808)
+
+### Issues fixed in this release
+
+**Bugs:**
+
+* Searching in PMUI with Static / Sleet package sources causes OutOfMemoryException - [#8478](https://github.com/NuGet/Home/issues/8478)
+
+* Improve DOTNET_HOST_PATH environment variable handling - [#7438](https://github.com/NuGet/Home/issues/7438)
+
+* nuget.exe spec Generates a copyright Tag with Hard-Coded Text of Copyright yyyy Instead of $copyright$ - [#8696](https://github.com/NuGet/Home/issues/8696)
```

Comment:
```suggestion
* nuget.exe spec generates a copyright tag with hard-coded text of Copyright YYYY Instead of $copyright$ - [#8696](https://github.com/NuGet/Home/issues/8696)
```